Platform/TigerlakeBoardPkg/AcpiTables/Rtct/Rtct.act

```diff
@@ -0,0 +1,58 @@
+/** @file
+

```

Comment:
If it is empty table, consider constructing the table at runtime. Don't need the RTCT file in flash at all.   Refer to mAcpiTblTmpl() in AcpiInitLib.  I think we can add one API in AcpiInitLib to allow platform to append new ACPI table template into mAcpiTblTmpl so that the ACPI core code can help create the table at runtime.

Reply:
yes, it would be great if the AcpiInitLib could provide a generic way to let platform to append a new table. Then we don't need have RTCT ASL code at all.

Reply:
I have created a PR https://github.com/slimbootloader/slimbootloader/pull/1075  to allow this.   Please take a look.